app/code/community/Businessfactory/Roihuntereasy/Model/Cron.php

```diff
@@ -7,25 +7,33 @@ public function _construct()
         parent::_construct();
     }
 
-    public function test() {
-        Mage::log("Cron schedule running.", null, "cron.log");
+    /**
+     * Method start new feed creation process from cron schedule.
+     */
+    public function scheduleFeed($schedule) {
+        Mage::log(__METHOD__ . ": scheduled feed generation. ", null, 'cron.log');
+
+        $jobsRoot = Mage::getConfig()->getNode('crontab/jobs');
+        $jobConfig = $jobsRoot->{$schedule->getJobCode()};
+        $fileFormat = (string) $jobConfig->format;
```

Comment:
Is this compatible with schedules created by older version of plugin? (If this case is possible) Maybe you should define the default format.